### PR TITLE
Feature/uabot 213 update features for external users

### PR DESCRIPTION
**Pull request by vBrusentsov**
This pull request allows users who do not have secret kes to the API to test the bot, namely to connect to the Google API and to connect to the air alarms API, and instructions on using these features have also been added to the README

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -246,6 +246,27 @@ DEBUG=
 # @optional {boolean} = false
 DEBUG_MIDDLEWARE=
 
+##
+# A feature to disable calls to the Google API 
+# Used when the bot does not have access to Google credits.
+#
+# Used for both bot and server process. 🤖 🖥️
+#
```
[the remark this answers is not in the document]
Comment (vBrusentsov) [reply .env.template]: Thank`s I removed extra space

File: src/services/alarm.service.ts [hunk 2]
```diff
@@ -33,6 +33,12 @@ export class AlarmService {
   testAlarmInterval?: NodeJS.Timer;
 
   getStates(): Promise<AlarmStates> {
+    if (Boolean(environmentConfig.DISABLE_ALARM_API) === true) {
```
[the remark this answers is not in the document]
Comment (vBrusentsov) [reply src/services/alarm.service.ts]: I simplified the check

File: src/services/alarm.service.ts [hunk 3]
```diff
@@ -90,10 +96,12 @@ export class AlarmService {
    * Creates SSE subscription to Alarm API events
    * */
   subscribeOnNotifications(reason: string) {
-    this.disable(reason);
+    if (Boolean(environmentConfig.DISABLE_ALARM_API) === true) {
```
[the remark this answers is not in the document]
Comment (vBrusentsov) [reply src/services/alarm.service.ts]: And here I simplified the check

File: src/services/dynamic-storage.service.ts [hunk 4]
```diff
@@ -61,6 +61,9 @@ export class DynamicStorageService {
   }
 
   async init() {
+    if (Boolean(environmentConfig.DISABLE_GOOGLE_API) === true) {
```
[the remark this answers is not in the document]
Comment (vBrusentsov) [reply src/services/dynamic-storage.service.ts]: And here I simplified the check

File: src/types/environment.ts [hunk 5]
```diff
@@ -44,6 +45,7 @@ export interface PostgresEnvironmentConfig {
 }
 
 export interface GoogleEnvironmentConfig {
+  DISABLE_GOOGLE_API: boolean;
```
[the remark this answers is not in the document]
Comment (vBrusentsov) [reply src/types/environment.ts:48]: Thank you. 
I did according to the example of the entire document

File: .env.template (hunk 1 above)
[the remark this answers is not in the document]
Comment (DrSmile444) [reply .env.template]: Thank you!

File: src/services/alarm.service.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (DrSmile444) [reply src/services/alarm.service.ts]: Thank you!

File: src/services/dynamic-storage.service.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (DrSmile444) [reply src/services/dynamic-storage.service.ts]: Thanks!

**Review by DrSmile444 — APPROVED** [review]
Good pull request. Thank you for your dedication!!! ❤️

**Pull request merged**